klippa/src/lib.rs

```diff
@@ -62,6 +70,98 @@ const MAX_NESTING_LEVEL: u8 = 64;
 // See <https://github.com/googlefonts/fontations/issues/997>
 const MAX_GID: GlyphId = GlyphId::new(0xFFFFFF);
 
+// ref: <https://github.com/harfbuzz/harfbuzz/blob/021b44388667903d7bc9c92c924ad079f13b90ce/src/hb-subset-input.cc#L82>
+pub static DEFAULT_LAYOUT_FEATURES: [Tag; 72] = [
```

Comment:
I would normally just make this a static slice instead of an array but that's just me?

(I mean like:

```rust
pub static DEFAULT_LAYOUT_FEATURES: &[Tag] = &[ ..
```
)